packages/core/useStorage/index.md

```diff
@@ -56,8 +56,8 @@ const state = useStorage(
   { mergeDefaults: true } // <--
 )
 
-console.log(state.hello) // 'nihao', from storage
-console.log(state.greeting) // 'hello', from merged default value
+console.log(state.value.stats) // 'nihao', from storage
```

Comment:
```suggestion
console.log(state.value.hello) // 'nihao', from storage
```

What's `stats`?

Reply:
I'm sorry, I have written the wrong word. I have revised the PR according to your suggestion.